modules/influxdb/influxdb.go

```diff
@@ -57,39 +72,24 @@ func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustom
 			// "Open shard" which is the last thing that happens before the server is ready to accept connections.
 			// This is probably different for InfluxDB 2.x, but that is left as an exercise for the reader.
 			strategies := []wait.Strategy{
-				genericContainerReq.WaitingFor,
+				wait.ForListeningPort("8086/tcp"),
 				wait.ForLog("influxdb init process in progress..."),
 				wait.ForLog("Server shutdown completed"),
 				wait.ForLog("Opened shard"),
 			}
-			genericContainerReq.WaitingFor = wait.ForAll(strategies...)
-			hasInitDb = true
-			break
+			return wait.ForAll(strategies...)
```

Comment:
Question: do we want to return here, without processing/appending the wait for log added by the image version tag?

Reply:
I chose not to append it to maintain consistency with the previous implementation. However, I’m uncertain about the best approach here, as I don’t have experience with InfluxDB 1.x or `docker-entrypoint-initdb.d` files. I would appreciate input from someone more familiar with these areas to help provide a clearer solution.